### PR TITLE
Build the dev container on each merge, and pull it with run-dev.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
This will mean we don't have to broadcast "make sure to update your container" every time it's needed.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: bin/build-dev [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+set +x
+export AWS_DEFAULT_REGION=us-west-2
+REGION=us-west-2
+
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/t1q6b4h2
+docker build -t uat-dev --cache-from public.ecr.aws/t1q6b4h2/universal-application-tool:latest .
```
Comment (hmhsu) [inline bin/build-dev:9]: Question (blocking): Does this need to be `public.ecr.aws/t1q6b4h2/uat-dev`?

File: .github/workflows/push.yaml [hunk 2]
```diff
@@ -14,6 +14,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Run build-dev.
+        id: build_dev
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
```
Comment (hmhsu) [inline .github/workflows/push.yaml:21]: Question (non-blocking): Do we need to login to do the push?

**Review by hmhsu — APPROVED** [review]
Looks good. Thanks for doing this.

File: bin/build-dev (hunk 1 above)
Comment (ndmckinley) [reply bin/build-dev:9]: Either would work but your suggestion will work better if / as the two diverge!  Changed.

File: .github/workflows/push.yaml (hunk 2 above)
Comment (ndmckinley) [reply .github/workflows/push.yaml:21]: Yep.

**Pull request merged**